database/src/main/java/dev/msfjarvis/lobsters/model/SubmitterAdapter.kt

```diff
@@ -0,0 +1,17 @@
+package dev.msfjarvis.lobsters.model
+
+import com.squareup.moshi.Moshi
+import com.squareup.sqldelight.ColumnAdapter
+
+class SubmitterAdapter : ColumnAdapter<Submitter, String> {
```

Comment:
I find it weird that we need this for `Submitter` but not `KeybaseSignature`. Can you check why?

Reply:
Looks like our SQLDelight model is not using `KeybaseSignature`. I'll fix it.

Reply:
Nah it's not that. `KeybaseSignature` is part of `Submitter`, and its serialization is covered by the JSON adapter generated for `Submitter`.

Reply:
Oh ohkay, I just took a quick look on the generated model.